### ISSUE TITLE
Any plans to configure ambari.properties?

### ISSUE THREAD

**Issue by holmesb** [jp/ambari#30]
Hi, 

Ambari Server's configuration is stored in /etc/ambari-server/conf/ambari.properties.  For example, to [enable ldap authentication](https://ambari.apache.org/1.2.1/installing-hadoop-using-ambari/content/ambari-chap2-4.html) requires changing numerous settings in this file.  Would be great for ambari.properties to be configurable by this cookbook.  Many thanks.

**Comment by wolf31o2** [comment]
Currently, we configure using `ambari-server setup -s` to create the properties file. You're looking, instead, for a fully-configurable template?

**Comment by holmesb** [comment]
Yes think so.  HortonWorks confirmed [ambari.properties will not be configured by BluePrints](https://community.hortonworks.com/questions/52322/configure-ambariproperties-using-blueprint.html), since they're intended for an operational server that already has ambari.properties defined.
